Objects/typeobject.c

```diff
@@ -4707,6 +4707,9 @@ static PyMethodDef object_methods[] = {
     {0}
 };
 
+PyDoc_STRVAR(object_doc,
+"object()\n--\n\nThe starting base class of all types and classes other\n"
+"than itself.");
```

Comment:
The first line of a docstring, if not too long, should not have an embedded \n.  So I think this should have been written

"object()\n--\n\n"
"The starting base class of all types and classes other than itself\n"

My suggested replacement would then be

"object()\n--\n\n"
"The base class of the class hierarchy.\n\n"
"When called, it accepts no arguments and returns a new featureless\n"
"instance that has no instance attributes and cannot be given any.\n"

A limit of 72 columns with a 4 column indent allows 68 chars.  The longest above is 66.